docs/resources/ocean_gke_launch_spec.md

```diff
@@ -84,6 +84,19 @@ resource "spotinst_ocean_gke_launch_spec" "example" {
         memory_per_unit = 2048
     }
   }
+  
+  network_interface {
+    network = "test-vng-network"
+    project_id = "test-vng-network-project"
+    access_configs {
+      access_configs_name = "external-nat-vng"
```

Comment:
does not match with actual variable name used in TF, change it accordingly

Reply:
done